packages/manager/modules/hub/src/components/user-panel/shortcuts/controller.js

```diff
@@ -18,31 +18,37 @@ export default class ManagerHubShortcutsCtrl {
         id: 'services',
         icon: 'oui-icon-multi-device_concept',
         url: this.RedirectionService.getURL('services'),
+        tracking: 'hub::sidebar::shortcuts::go-to-services',
       },
       {
         id: 'bills',
         icon: 'oui-icon-receipt_concept',
         url: this.RedirectionService.getURL('billing'),
+        tracking: 'hub::sidebar::shortcuts::go-to-bills',
       },
       {
-        id: 'assistance',
+        id: 'supportLevel',
```

Comment:
No link to the `assistance` part of the manager ? Or you access it through the `service level` part

Reply:
there'll be the support tile for that